switchbot/const.py

```diff
@@ -71,3 +72,35 @@ class LockStatus(Enum):
     LOCKING_STOP = 4  # LOCKING_BLOCKED
     UNLOCKING_STOP = 5  # UNLOCKING_BLOCKED
     NOT_FULLY_LOCKED = 6  # LATCH_LOCKED - Only EU lock type
+
+
+class HumidifierMode(Enum):
+    HIGH = 1
+    MEDIUM = 2
+    LOW = 3
+    QUIET = 4
+    TARGET_HUMIDITY = 5
+    SLEEP = 6
+    AUTO = 7
+    DRYING_FILTER = 8
+
+
+class HumidifierWaterLevel(Enum):
+    EMPTY = 0
+    LOW = 1
+    MEDIUM = 2
+    HIGH = 3
+
+
+OVER_HUMIDIFY_PROTECTION_MODES = {
+    HumidifierMode.QUIET,
+    HumidifierMode.LOW,
+    HumidifierMode.MEDIUM,
+    HumidifierMode.HIGH,
+}
+
+
+TARGET_HUMIDITY_MODES = {
+    HumidifierMode.SLEEP,
+    HumidifierMode.TARGET_HUMIDITY,
+}
```

Comment:
I'd keep these in the humidifier module since they aren't used outside of it and nothing else needs them AFAICT

Reply:
Moving them there causes circular dependency issues, how about separate `const_*.py` or `const/*.py` files for different devices?

Reply:
Works for me

Reply:
Sorry, I'm not sure if you meant it as moving them to `switchbot/devices/evaporative_humidifier.py` works for you or you agree with with separate const files?

Reply:
Separate constant or models files is fine